rai/client.go

```diff
@@ -1713,3 +1713,67 @@ func (c *Client) ListSnowflakeDatabaseLinks(
 	}
 	return result, nil
 }
+
+//
+// Snowflake Data Steams
+//
+
+func (c *Client) CreateSnowflakeDataSteam(
+	integration, database, schema, object, role, warehouse string, isView bool,
+	raiDatabase, relation string,
+	columns []SnowflakeSchemaColumn, creds *SnowflakeCredentials,
+) (*SnowflakeDataStream, error) {
+	var result SnowflakeDataStream
+	dbLink := fmt.Sprintf("%s.%s", database, schema)
+	objectName := fmt.Sprintf("%s.%s", dbLink, object)
+	path := makePath(PathIntegrations, integration, "database-links", dbLink, "data-streams")
+	req := createSnowflakeDataStreamRequest{}
+	req.Snowflake.Object = objectName
+	req.Snowflake.IsView = isView
+	req.Snowflake.Columns = columns
+	req.Snowflake.Role = role
+	req.Snowflake.Warehouse = warehouse
+	req.Snowflake.Credentials = *creds
+	req.RAI.Database = raiDatabase
+	req.RAI.Relation = relation
+	if err := c.Post(path, nil, &req, &result); err != nil {
+		return nil, err
+	}
+	return &result, nil
+}
+
+func (c *Client) DeleteSnowflakeDataSteam(
+	integration, database, schema, object, role string, creds *SnowflakeCredentials,
+) error {
+	dbLink := fmt.Sprintf("%s.%s", database, schema)
+	objectName := fmt.Sprintf("%s.%s", dbLink, object)
```

Comment:
the object name is expected to be fully qualified already, as it may not be in the same db/schema of the dblink

Reply:
same applies to the other 3 functions